.github/workflows/kind-e2e.yaml

```diff
@@ -22,47 +22,47 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - name: skip test
-      # see https://github.com/knative-sandbox/kn-plugin-quickstart/issues/392
-      run: |
-        echo "github action runner broke this test"
-
-    # - name: Set up Go 1.18.x
-    #   uses: actions/setup-go@v3
-    #   with:
-    #     go-version: 1.18.x
-    # 
-    # - name: Build quickstart plugin
-    #   run: |
-    #     set -x
-    #     ./hack/build.sh
-    #     sudo mv kn-quickstart /usr/local/bin
-    # 
-    # - name: Install Dependencies
-    #   run: |
-    #     set -x
-    #     echo "::group:: install kind ${KIND_VERSION}"
-    #     curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/v${KIND_VERSION}/kind-linux-amd64
-    #     chmod +x ./kind
-    #     sudo mv kind /usr/local/bin
-    #     echo "::endgroup::"
-    # 
-    #     echo "::group:: install kn client ${KNATIVE_VERSION}"
-    #     curl -Lo ./kn https://github.com/knative/client/releases/download/knative-v${KNATIVE_VERSION}/kn-linux-amd64
-    #     chmod +x ./kn
-    #     sudo mv kn /usr/local/bin
-    #     echo "::endgroup::"
-    # 
-    # - name: Serving e2e Test
+    # - name: skip test
+    #   # see https://github.com/knative-sandbox/kn-plugin-quickstart/issues/392
     #   run: |
-    #     kn quickstart kind --install-serving
-    #     source ./vendor/knative.dev/hack/e2e-tests.sh
-    #     ./test/serving-e2e-test.sh || fail_test
-    # 
-    # - uses: chainguard-dev/actions/kind-diag@main
-    #   # Only upload logs on failure.
-    #   if: ${{ failure() }}
-    #   with:
-    #     cluster-resources: nodes,${{ matrix.cluster-resources || '' }}
-    #     namespace-resources: pods,svc,ksvc,route,configuration,revision,king,${{ matrix.namespace-resources || '' }}
-    #     artifact-name: logs-${{ matrix.k8s-version}}-${{ matrix.ingress }}-${{ matrix.test-suite }}
+    #     echo "github action runner broke this test"
+
+    - name: Set up Go 1.18.x
```

Comment:
This probably needs a bump to v1.20 (which I think is what we're building the rest of the project with)